MediaStreamTrack.js

```diff
@@ -83,6 +83,16 @@ export default class MediaStreamTrack extends EventTarget(MEDIA_STREAM_TRACK_EVE
     this.muted = !this._enabled;
   }
 
+  _switchCamera() {
```

Comment:
I do not claim that it's always been the case of the contributions of the Jitsi community but I myself made conscious efforts to at least try to have documentation comments in what I propose for merging into oney/react-native-webrtc. So please take it with many grains of salt. I believe in this case a jsdoc will be very useful because you're introducing non-standard JS WebRTC API. Something like why we decided to go about this way. You pretty much have it in the PR description, I just feel it'd be useful in the source code itself as well.